core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperators.java

```diff
@@ -172,6 +172,13 @@ private SqlLibraryOperators() {
   @LibraryOperator(libraries = {MYSQL, ORACLE})
   public static final SqlFunction REGEXP_REPLACE = new SqlRegexpReplaceFunction();
 
+  @LibraryOperator(libraries = {MYSQL})
+  public static final SqlFunction EXTRACT_VALUE = new SqlFunction(
+      "EXTRACTVALUE", SqlKind.OTHER_FUNCTION,
+      ReturnTypes.cascade(ReturnTypes.VARCHAR_2000, SqlTypeTransforms.FORCE_NULLABLE),
+      null, OperandTypes.STRING_STRING, SqlFunctionCategory.SYSTEM);
+
+
```

Comment:
Oracle also has this function[1]. Is it the same?

[1] https://docs.oracle.com/cd/B19306_01/server.102/b14200/functions052.htm

Reply:
Reading the docs, seems like there is difference in behaviour of MySql ExtractValue and Oracle ExtractValue.